### PR TITLE
Changes the main window's text/title to ASB namespace.

### LINKED ISSUE #523: Switch main window text/name/header to current connection ASB namespace.
At the moment of the issue when I have multiple SBE windows opened at the same time and connected to 3 (basically more than 1) ASB instances, I can't really tell the difference between them by looking at the windows. Either during task switching (like Alt-Tab shortcut on Windows) or using gestures on Touchpad or when hovering over the SBE icons in the taskbar. 

See below for screenshots.
image
image

Can we switch the main title/text of the window to ASB namespace when the connection is loaded/made?

### PR DESCRIPTION
**Pull request by groozin**
This brings more visibility when working with mulitple ASB connections at the same time.

Closes #523

### CODE REVIEW

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 5.1.0-alpha.19.build.803 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/38259448) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/e63642e810 by @groozin)

File: src/ServiceBusExplorer/Forms/MainForm.cs [hunk 1]
```diff
@@ -339,6 +339,7 @@ private void SavedConnectionToolStripMenuItem_Click(object sender, EventArgs e)
         {
             var serviceBusNamespace = serviceBusHelper.ServiceBusNamespaces[(sender as ToolStripMenuItem).Tag.ToString()];
             serviceBusHelper.Connect(serviceBusNamespace);
+            this.Text = $"{serviceBusNamespace.Namespace}";
```
Comment (SeanFeldman) [inline src/ServiceBusExplorer/Forms/MainForm.cs:342]: Is string interpolation necessary here?

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 5.1.0-alpha.19.build.804 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/38265031) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/fe719ba646 by @groozin)

File: src/ServiceBusExplorer/Forms/MainForm.cs (hunk 1 above)
Comment (groozin) [reply src/ServiceBusExplorer/Forms/MainForm.cs]: Nope. It is leftover from the experimentation with the text to display. Fixed in the subsequent commit now.

**Review by ErikMogensen — CHANGES_REQUESTED** [review]
I think the title should be "<namespace> - Service Bus Explorer". That aligns with Notepad and VS Code which has the same behavior. 

Then it would be best to put the setting of the title in a new method.

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 5.1.0-alpha.19.build.805 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/38279037) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/f85f2dadc2 by @groozin)

**Review by SeanFeldman — APPROVED** [review]

**Review by ErikMogensen — APPROVED** [review]
Great!

**Comment by paolosalvatori** [comment]
Thanks for the PR @groozin 😄 what is the new format for the title now? Namespace - Service Bus Explorer?

**Pull request merged**